tests/system/drop_unsampled_transactions.approved.json

```diff
@@ -124,9 +126,9 @@
             }
         },
         "observer": {
-            "ephemeral_id": "ebd89158-9f85-469e-9bc0-9e19b174a1ba",
+            "ephemeral_id": "d0a481f0-3688-40dc-8a59-04abdf72261a",
             "hostname": "goat",
-            "id": "01019c57-ff25-4ddf-b9e9-2f421577a9a2",
+            "id": "fb7e85d9-3666-4b9a-8440-1448c17b5738",
```

Comment:
why all these changes? `id, `ingested`, etc?

Reply:
They get automatically updated by the approvals tests.